datalake/src/main/java/com/sequenceiq/datalake/configuration/CDPConfig.java

```diff
@@ -42,12 +47,14 @@
                 Matcher matcher = Pattern.compile(".*/runtime/(.*)/(.*)/(.*).json").matcher(resource.getURL().getPath());
                 if (matcher.find()) {
                     String runtimeVersion = matcher.group(RUNTIME_GROUP);
-                    CloudPlatform cloudPlatform = CloudPlatform.valueOf(matcher.group(CLOUDPLATFORM_GROUP).toUpperCase());
-                    SdxClusterShape sdxClusterShape = SdxClusterShape.valueOf(matcher.group(CLUSTERSHAPE_GROUP).toUpperCase());
-                    CDPConfigKey cdpConfigKey = new CDPConfigKey(cloudPlatform, sdxClusterShape, runtimeVersion);
-                    String templateString = IOUtils.toString(resource.getInputStream(), StandardCharsets.UTF_8.name());
-                    StackV4Request stackV4Request = JsonUtil.readValue(templateString, StackV4Request.class);
-                    cpdStackRequests.put(cdpConfigKey, stackV4Request);
+                    if (!unsupportedRuntimes.contains(runtimeVersion)) {
```

Comment:
Is it supposed to be a comma-separated list? If so then contains can match partially.